server/ladder_service.py

```diff
@@ -500,6 +501,11 @@ def make_game_options(player: Player) -> GameLaunchOptions:
                     "Ladder game failed to start! %s setup timed out",
                     game
                 )
+            elif isinstance(e, GameClosedError):
+                self._logger.info(
+                    "Ladder game %s failed to start! Player %s closed their game instance",
+                    game, e.player
```

Comment:
When I get around to finishing the matchmaker ban for leaving game, I will use this to register a violation